include/z64actor.h

```diff
@@ -100,6 +100,38 @@ typedef struct {
     /* 0x18 */ Vec3f feetPos[2]; // Update by using `Actor_SetFeetPos` in PostLimbDraw
 } ActorShape; // size = 0x30
 
+#define ACTOR_FLAG_NONE 0
```

Comment:
Two weird spots that probably were the reason I added _NONE:

https://github.com/Dragorn421/oot/blob/09c8d4f3b6526ceb08b164622a32d230c46f0f4b/src/overlays/actors/ovl_En_Fr/z_en_fr.c#L229
```c
this->actor.flags &= ~ACTOR_FLAG_NONE;
```

https://github.com/Dragorn421/oot/blob/09c8d4f3b6526ceb08b164622a32d230c46f0f4b/src/overlays/actors/ovl_En_Skj/z_en_skj.c#L387
```c
this->actor.flags |= ACTOR_FLAG_NONE;
```

what about those three (two in en_skj) spots, just use 0?

other than that I agree `#define FLAGS ACTOR_FLAG_NONE` should really be `#define FLAGS 0`

Reply:
For these cases, we have quite a bit of freedom and there is no way to tell what it was originally.
I think they can all be:

A)
```c
flags &= ~foo;
flags |= 0;
```
B)
```c
flags &= ~foo;
flags &= ~0;
```
C)
```c
flags &= ~foo;
flags = flags;
```
D) (fairly likely what it was, due to a typo with &= instead of &)
```c
flags = flags &= ~foo;
```
And possibly others like ^= 0

I don't know which option is better, but I definitely don't think we should introduce a define for them either way